src/explore-education-statistics-frontend/src/modules/table-tool/components/TableToolFinalStep.tsx

```diff
@@ -119,38 +121,50 @@ const TableToolFinalStep = ({
             ref={dataTableRef}
             fullTable={table}
             tableHeadersConfig={currentTableHeaders}
+            onError={message => {
+              toggleHasTableError.on();
+              logEvent({
+                category: 'Table Tool',
+                action: 'Table rendering error',
+                label: message,
+              });
+            }}
           />
         </>
       )}
-      <div className="govuk-!-margin-bottom-7">
-        <TableToolShare
-          tableHeaders={currentTableHeaders}
-          query={query}
-          selectedPublication={selectedPublication}
-        />
-      </div>
+      {!hasTableError && (
```

Comment:
Is there any momentary flicker where this displays before the table error is surfaced?

Reply:
Not that I can see.

Reply:
Cool, I guess a slight flicker would be acceptable anyway. Approved!